src/Raven.Server/RavenServer.cs

```diff
@@ -276,24 +276,30 @@ public void RefreshClusterCertificate(object state)
         {
             try
             {
-                CertificateHolder newCertificate = null;
+                CertificateHolder newCertificate;
                 try
                 {
                     newCertificate = LoadCertificate();
+                    if (newCertificate == null)
+                    {
+                        if (Logger.IsOperationsEnabled)
+                            Logger.Operations("Tried to load certificate as part of refresh check, and got a null back, but got a valid certificate on startup!");
+
+                        ServerStore.NotificationCenter.Add(AlertRaised.Create(
+                            null,
+                            "Server certificate",
+                            "Tried to load certificate as part of refresh check, and got a null back, but got a valid certificate on startup!",
+                            AlertType.Certificates_ReplaceError,
+                            NotificationSeverity.Error,
+                            "Cluster.Certificate.Replace.Error"));
+                        return;
+                    }
                 }
                 catch (Exception e)
                 {
-                    if (Logger.IsOperationsEnabled)
-                        Logger.Operations("Tried to load certificate as part of refresh check, but got an error!", e);
+                    throw new InvalidOperationException("Tried to load certificate as part of refresh check, but got an error!", e);
```

Comment:
Why aren't we logging this?

Reply:
`LoadCertificate` can error other than returning null,no?

Reply:
We are logging + alerting, in the outer try catch